README.md

```diff
@@ -169,6 +169,27 @@ You can import word templates by adding `<documenttemplate>` elements.
 </templateconfig>
 ```
 
+### Attribute specific functionality (Auto-number seed values)
```

Comment:
I think the table of contents needs to be updated to include these headers.

Reply:
Good catch, updated